panda/src/x11display/x11GraphicsWindow.cxx

```diff
@@ -2051,7 +2051,7 @@ get_keyboard_map() const {
       continue;
     }
 
-    KeySym sym = XKeycodeToKeysym(_display, k, 0);
+    KeySym sym = XkbKeycodeToKeysym(_display, k, 0, 0);
```

Comment:
Hmm, are you absolutely sure that this doesn't require linking with an additional library?  That's the main reason I was reluctant to make this change.